packages/examples/automatic_tracing/README.md

```diff
@@ -0,0 +1,82 @@
+# OpenCensus Node.js Automatic Tracing
```

Comment:
Does it make more sense to put `examples` at the top level rather than in `packages`?

Reply:
Yep. Actually, as we are planning to leave the more technical README at `@opencensus/nodejs` maybe we should put `examples` there. What do you think?

Reply:
I have a slight preference for the root directory. Either is better than having it in `packages`.

Reply:
Maybe in root makes it more visible. I'm going to change to that.